Packs/QuestKace/Integrations/QuestKace/QuestKace.py

```diff
@@ -0,0 +1,785 @@
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        return self._http_request("GET", url_suffix=f"/service_desk/queues/{queue_number}/fields", headers=headers)
+
+    def tickets_list_request(self, shaping_fields: str = None, filter_fields: str = None) -> dict:
+        """List of Tickets.
+            Args:
+                shaping_fields: str of the shaping that will be sent in the request.
+                filter_fields: str of filter that will be sent in the request.
+           Returns:
+               Response from API.
+        """
+        url = "/service_desk/tickets"
+        if not shaping_fields:
+            shaping_fields = set_shaping(self)
+        self.update_token()
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie
+        }
+        if filter_fields:
+            return self._http_request("GET", url_suffix=f"{url}?filtering={filter_fields}&"
+                                                        f"shaping={shaping_fields}", headers=headers)
+        else:
+            return self._http_request("GET", url_suffix=f"/service_desk/tickets?shaping={shaping_fields}",
+                                      headers=headers)
+
+    def create_ticket_request(self, data: str) -> dict:
+        """Create Ticket
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("POST", url_suffix="/service_desk/tickets", headers=headers, data=data)
+
+    def update_ticket_request(self, ticket_id: str, data: str) -> dict:
+        """Update Ticket.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("POST", url_suffix=f"/service_desk/tickets/{ticket_id}", headers=headers, data=data)
+
+    def delete_ticket_request(self, ticket_id: str) -> dict:
+        """Delete Ticket.
+           Returns:
+               Response from API.
+        """
+        headers = {
+            'Accept': 'application/json',
+            'x-dell-csrf-token': self._token,
+            'x-dell-api-version': '5',
+            'Cookie': self._cookie,
+            'Content-Type': 'application/json'
+        }
+        return self._http_request("DELETE", url_suffix=f"/service_desk/tickets/{ticket_id}", headers=headers)
+
+    def ticket_by_id_request(self, id: int) -> dict:
```

Comment:
is not `id` a saved word?
try adding `_` to every saved word you want to use as a variable, e.g: `id_`